### PR TITLE
Portfolio header

### PR DESCRIPTION
**Pull request by teckim**
in this pull request:

- Added the base structure for the portfolio project.
- Added Main nav bar.
- Added Headline.
- Added Linters.

> Note: the stylelint didn't pass the checks even though it did pass in my local machine. 
According to the error details the problem is with stylelint itself. 
I would be glad to know if there's a solution for this issue, knowing that I already tried to install `postcss`

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,9 +5,54 @@
...
+  <body>
+    <header class="header">
+      <nav class="toolbar">
+        <div class="toolbar__logo-wrapper">
+          <span class="toolbar__logo">Hakim</span>
+        </div>
+        <button class="toolbar__menu-button" type="button">
+          <img src="./assets/icons/menu.svg" alt="Menu icon" />
+        </button>
+      </nav>
```
Comment (m05e5) [inline src/index.html]: * You did a very great job with the UI 
  - [ ]  As mentioned in the requirements, the logo should be wrapped in a < `a` > tag that redirects to the home page(in this case its the same page).
  - [x] The logo should be out of the `nav`, but should still be in the header. Please kindly fix that.

File: src/index.html [hunk 2]
```diff
@@ -5,9 +5,54 @@
...
+          </a>
+          <a>
+            <img src="./assets/icons/linkedin.svg" alt="" />
+          </a>
+          <a>
+            <img src="./assets/icons/github.svg" alt="" />
+          </a>
+        </div>
+      </div>
+    </header>
```
Comment (m05e5) [inline src/index.html]: - [x] Please the rest of the page content should be in the body but not in the header. Please kindly move it out of the header and change the `div` to `section`. By doing so your code will look more professional and more semantic. lets work like experts 😃

File: src/index.html [hunk 3]
```diff
@@ -5,9 +5,54 @@
...
+          <a>
+            <img src="./assets/icons/twitter.svg" alt="" />
+          </a>
+          <a>
+            <img src="./assets/icons/linkedin.svg" alt="" />
+          </a>
+          <a>
+            <img src="./assets/icons/github.svg" alt="" />
+          </a>
+        </div>
```
Comment (m05e5) [inline src/index.html]: - [x] Please the list of icons uses a `ul` tag to group them, and `li` per icon

**Review by m05e5 — COMMENTED** [review]
Hi @teckim , 

Good job so far. I really like what you did!
There are some minor issues that you still need to work on to go to the next project but you are almost there!
mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️

## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve
  - [ ] For the stylint issue I strongly recommend you reinstall the linters completely.  [Here ](https://github.com/microverseinc/linters-config/tree/master/html-css)is the link to the linter setup
  
  _For the rest please check the comments under this review_

_I'm sure you can do it just keep it up_
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (teckim) [reply src/index.html]: @m05e5 thank you for your comment, please I would like to know where is the requirement that says logo should be an anchor (`<a>`) tag?

**Comment by teckim** [comment]
> Hi @teckim ,
> 
> Good job so far. I really like what you did! There are some minor issues that you still need to work on to go to the next project but you are almost there! mario_ladders
> 
> ## Good points 👍
> You used proper GitHub Flow ✔️ Your linter checks are all passing ✔️
> 
> ## Highlights
> ## Required Changes ♻️
> * You really did a great job but there are some features you need to improve
>   
>   * [ ]  For the stylint issue I strongly recommend you reinstall the linters completely.  [Here ](https://github.com/microverseinc/linters-config/tree/master/html-css)is the link to the linter setup
>   
>   _For the rest please check the comments under this review_
> 
> _I'm sure you can do it just keep it up_
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> _@m05e5_
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hi @m05e5, thank you for your valuable review on my work, I tried everything to fix the linter issue but no success, it's hard to analyze the issue since it happens on the github server when it tries to run the stylelint command. Please tell me what should I do? 
thank you.

File: src/index.html [hunk 4]
```diff
@@ -5,9 +5,63 @@
...
-  <body></body>
+  <body>
+    <header class="header">
+      <div class="toolbar">
+        <div class="toolbar__logo-wrapper">
+          <a href="/" class="toolbar__logo">Hakim</a>
+        </div>
+        <button class="toolbar__menu-button" type="button">
+          <img src="./assets/icons/menu.svg" alt="Menu icon" />
+        </button>
```
Comment (ShahierNashaat) [inline src/index.html]: - [ ] Please put the hamburger button inside a `nav` tag.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @teckim , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by teckim** [comment]
Hi @m05e5 again!
So I managed to fix the issue by adding `postcss@8.x` to the list of dependencies in the linter config file.

**Review by zhadier — APPROVED** [review]
# Hi @teckim 👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Great job implementing the required changes from the previous reviewer 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**